Dokument/Dateien/Magie_Essenz.tex

```diff
@@ -1,103 +1,876 @@
+\end{outline}
+Zurück zu Tab. \ref{tab:magie-verbreitung}.
+
+
+
+\subsection{Röntgen}\label{magie:roentgen}
+Röntgenstrahlung erlaubt es, tief in organisches Material hineinzusehen. 
+
+\subsubsection{Verbreitung}
+Einige Tiere beherrschen diese Fähigkeit. \\
+Keine Verbreitung unter den Hominini.
+
+\subsubsection{Bedeutung im Spiel}
+Derzeit keine Bedeutung im Spiel.
+
+\subsubsection{Intuitive Nutzung}
+Gerade Jäger, die ihre Beute unter Deckung wie Felsen oder Erde suchen, nutzen kurze blitzartige Momente der Röntgenstrahlung, um ihre Beute zu lokalisieren.
+Da diese Strahlung sehr energieintensiv ist, ist eine häufige Anwendung mit dem normalen Pool an Mana nicht möglich.
+\\ \\
+Zurück zu Tab. \ref{tab:magie-verbreitung}.
+
+
+
+\subsection{Rundfunk}\label{magie:rundfunk}
+Diese Wellenlänge wird heutzutage zum Rundfunk genutzt.
+Uns ist nichts eingefallen, wie man dies als Fähigkeit für eine nicht-technologisierte Welt einsetzen könnte.
+Denn den Wesen einfach einen (detaillierten) Interpreter und Sender wie bei unseren Radiostationen in den Kopf zu packen, erschien uns unlogisch.
+Da diese Frequenzen aber mitten drin unter den anderen genutzten liegt, ist eine Evolution dieser Fähigkeit äußerst wahrscheinlich.
+Daher behalten wir sie uns als Option drin, insbesondere für eine spätere Nutzung im entsprechenden Zeitalter.
+
+Sollte jemand sich mit der Thematik des Rundfunks besser auskennen und doch dafür plädieren (und es damit als eine Art Telepathie einsetzen wollen), dann bitte zur Diskussion melden.
+
+\subsubsection{Verbreitung}
+Keine Verbreitung unter den Lebewesen.
+
+\subsubsection{Bedeutung im Spiel}
+Derzeit keine Bedeutung im Spiel.
+\\ \\
+Zurück zu Tab. \ref{tab:magie-verbreitung}.
+
+
+
+\subsection{Schub}\label{magie:schub}
+Schub steht für die mechanische Energie, die auf etwas appliziert werden kann.
+Diese Magieart entspricht damit dem Bild einer erweiterten Hand, die diese Kraft ausübt.
+
+
+Zur Unterscheidung von \nameref{magie:druck} kann man sich hier vor Augen halten, dass der Druck auf alles an der entsprechenden Stelle gilt, da er außerhalb angesetzt wird (das kann auch Auswirkungen auf das Objekt selbst haben!). 
+Die mechanische Kraft wirkt jedoch nur auf ein bestimmtes Objekt.
+Damit sind bei ähnlichen Anwendungen präzisere Ausführungen möglich.
+Zudem kann je nach Umstand der Druck an seine Grenzen kommen, wo die mechanische Kraft noch keine Grenzen hat.
+
+Weiterhin unterscheidet sie sich vom Druck darin, dass es immer um eine Bewegung geht.
+Kann mittels Druck etwas zerquetscht werden, so ist dies mit Schub nicht möglich: dazu müsste man andere Dinge nehmen, die den Schub Richtung des zu Zerquetschendes erhalten.
+
+\subsubsection{Verbreitung}
+Eine normale, weit verbreitete Fähigkeit.
+
+\subsubsection{Bedeutung im Spiel}
+Eine normale Fähigkeit unter der Bevölkerung.
+
+\subsubsection{Intuitive Nutzung}
+Ideen pls.
```

Comment:
+ evtl Abfedern von Einschlägen/Aufstößen, indem der Druck im Körper verteilt wird anstatt lokal zu wirken

Reply:
Aber das ist ja wieder eine Druck-Geschichte... 
Andererseits könnte man es anders nutzen: was wäre, wenn diejenigen sich selbst kurz vor Ankommen auf dem Boden (beim Springen) einen mechanischen Schub nach oben geben und dadurch weiter fallen können, da sie sich nicht so schnell verletzen? Also ist ja immer noch intuitives Level und daher kann nicht zu viel Magie auf einmal gewirkt werden. Also vom "Hochhaus" springen ist halt immer noch tödlich :D

Reply:
so in etwa meinte ich das auch ^^